twisted/internet/cfreactor.py

```diff
@@ -474,9 +478,9 @@ def install(runLoop=None, runner=None):
     @param runLoop: the run loop to use.
 
     @param runner: the function to call in order to actually invoke the main
-        loop.  This will default to L{CFRunLoopRun} if not specified.  However,
-        this is not an appropriate choice for GUI applications, as you need to
-        run NSApplicationMain (or something like it).  For example, to run the
+        loop.  This will default to L{CoreFoundation.CFRunLoopRun} if not specified.
+        However, this is not an appropriate choice for GUI applications, as you need
+        to run NSApplicationMain (or something like it).  For example, to run the
         Twisted mainloop in a PyObjC application, your C{main.py} should look
```

Comment:
OK.  @adiroiban submitted a similar fix to add zope links https://github.com/twisted/twisted/pull/238
Do you know what link we would need to add in a similar way for CoreFoundation and CFNetwork?